noscrypto/Client.py

```diff
@@ -58,3 +58,13 @@ def WorldEncrypt(packet, session, is_first_packet = False):
     
 def WorldDecrypt(packet):
     return Utils._Unpack(packet, _DECRYPTION_TABLE)
+
+def WorldDecryptList(packets_raw, encoding="ascii"):
+    out = []
+    for this_packet in packets_raw.split(b'\xff'):
+        if this_packet == b'':

```

Comment:
The comparison could be simplified to `if not this_packet:`

Reply:
Thanks for contribution, I was against this change, because of how the TCP protocol work (it doesn't always send complete packets each time). However you are probably third person asking for such a function, so maybe it would be worth to implement.